### PR TITLE
Change SettingsOption icon to Splits icon in DataTable example

### LINKED ISSUE #2087: Ordering of Items - Update site examples replacing cog icon with column icon
Related https://github.com/grommet/hpe-design-system/issues/2063

- [x] Update the example on [DataTable Customization](https://design-system.hpe.design/templates/datatable-customization) to use column icon instead of cog icon.
- [x] Double-check that there are no other uses of cog icon in the site's examples.

[Figma](https://www.figma.com/file/TonE8uQ33e5QJRPknZx36q/Ordering-Of-Items?node-id=868%3A13552)

### PR DESCRIPTION
**Pull request by jcfilben**
[Deploy Preview](https://deploy-preview-2088--keen-mayer-a86c8b.netlify.app/)

#### What does this PR do?
Replaces the Cog (SettingsOption) Icon in the DataTable example. Research showed that the icon was confusing and the Splits icon made more sense to people.

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [x] Small, medium, and large screen sizes
- [x] Cross-browsers (FireFox, Chrome, and Safari)
- [x] Light & dark modes
- [x] All hyperlinks route properly

**Accessibility Checks**
- [x] Keyboard interactions
- [x] Screen reader experience
- [x] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [x] Console is free of warnings and errors
- [ ] Passes E2E commit checks - **I was getting lint errors when trying to merge with master. Looks like some of the changes accepted to master are not passing lint checks.**
- [x] Visual snapshots are reasonable

#### How should this be manually tested?
Check that the icon in the DataTable customization example is the Splits icon.

#### What are the relevant issues?
Closes #2087 

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible

### CODE REVIEW

File: aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ export const TableCustomizationExample = () => {
                   actions={
                     <Box direction="row" align="start" gap="small" flex={false}>
                       <DropButton
-                        icon={<SettingsOption />}
+                        icon={<Splits />}
```
Comment (halocline) [inline aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js:120]: We have other accessibility improvements we should do for once the drop opens, however, we should provide an a11yTitle here at a minimum.

```suggestion
                        a11yTitle="Configure columns, button"
                        icon={<Splits />}
```

Comment (halocline) [reply aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js:120]: We should also include `aria-labeledby="search-cols-icon"` in the [columns visibility example](https://deploy-preview-2088--keen-mayer-a86c8b.netlify.app/templates/datatable-customization?q=data#column-visibility)

I'm okay if we create an issue for "Fix accessibility issues on DataTable Customization page" if you'd prefer for the search icon and datatable checkbox WAVE errors.

**Review by halocline — COMMENTED** [review]
RE: Screen reader experience for the "configure columns" button, we should fix this.

RE:  Console is free of warnings and errors - There is an error in the console on the DataTable page but it was already present before this change.

Trying to practice "leave no trace" ethics with this codebase. If we spot something, even if unrelated to current PR, if within reason we should try to leave the codebase improved.

Would you be okay modifying the useEffect in ExampleResources.js to something like 
```
  React.useEffect(() => {
    const timer = setTimeout(() => {
      if (codeOpen) {
        fetchCode(activeCode);
      }
    }, 500);
    return () => {
      clearTimeout(timer);
    };
  }, [activeCode, codeOpen]);
  ```
  
  RE: other accessibility items on this page - let's either fix or create an issue.

**Comment by halocline** [comment]
P.S. Thank you for using the testing checklist - super helpful.

**Comment by jcfilben** [comment]
I like the 'leave no trace' idea

In the latest commit I fixed the useEffect error and the accessibility errors.

**Review by halocline — APPROVED** [review]

**Pull request merged**